registry/repositories.go

```diff
@@ -5,6 +5,11 @@ import (
 	"fmt"
 	"net/http"
 	"net/url"
+	"regexp"
+)
+
+var (
+	harborAPIv2Pattern = regexp.MustCompile(`\/api\/v2\.0\/projects`)
```

Comment:
Forward slashes in regular expressions don't need to be escaped.

```suggestion
	harborAPIv2Pattern = regexp.MustCompile(`/api/v2\.0/projects`)
```